bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/filters/TestTimestampRangeFilterAdapter.java

```diff
@@ -34,8 +34,8 @@
   @Test
   public void timestampFiltersAreAdapted() {
     TimestampRangeFilter filter = new TimestampRangeFilter(10L, 20L);
-    RowFilter rowFilter = filterAdapter.adapt(emptyScanContext, filter);
-    Assert.assertEquals(10000L, rowFilter.getTimestampRangeFilter().getStartTimestampMicros());
-    Assert.assertEquals(20000L, rowFilter.getTimestampRangeFilter().getEndTimestampMicros());
+    Filters.Filter expectedFilter = filterAdapter.adapt(emptyScanContext, filter);
+    Assert.assertEquals(10000L, expectedFilter.toProto().getTimestampRangeFilter().getStartTimestampMicros());
```

Comment:
please extract a variable for `expectedFilter.toProto().getTimestampRangeFilter()`